### PR TITLE
Add php-cs rules

### PR DESCRIPTION
**Pull request by Spea**
/cc @rebuy-de/prp-blue-web @rebuy-de/prp-green-web 

FYI: @rebuy-de/leader

### CODE REVIEW

**Comment by Spea** [comment]
I should have mentioned, that the coding styles changed from version 1.x to 2.x, thus I had to make some adjustments.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+### Travis code style check
+
+```bash
+vendor/bin/php-cs-fixer fix --config=vendor/rebuy/php-cs/.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only --diff-filter=ACMRTUXB $TRAVIS_COMMIT_RANGE`
+```
+
+### Fixing code styles
+
+```bash
+vendor/bin/php-cs-fixer fix --config=vendor/rebuy/php-cs/.php_cs.dist yourDirectory/`
```
Comment (sod) [inline README.md:18]: I like to do 

```bash
git diff --name-only master | grep ".php$" | while read -r file; do; echo "File: $file"; vendor/bin/php-cs-fixer fix --verbose --config-file=vendor/rebuy/php-cs/.php_cs.dist $file; done
```

that fixes all changed php files from `master` to current branch.

**Review by sod — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Spea) [reply README.md:18]: Will add another example then. But I also will adjust our own command for fixing coding styles, so you don't have to enter this whole string manually :)

Comment (Spea) [reply README.md:18]: Added another example below this one, which is way simpler and does not start the php-cs-fixer for every file :)

**Review by lukaszklis — APPROVED** [review]

**Pull request merged**